PersonalAccessTokenAPIAppSample/ms-identity-python-webapp-master/CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,9 @@
+# Microsoft Open Source Code of Conduct
```

Comment:
Can you move the code of conduct into the root of this repo? it's missing there so far